docs/kratos/session-management/01_overview.mdx

```diff
@@ -27,113 +30,82 @@ For security reasons, you can't break the isolation between cookies and session
-  }
-}
-```
+An example of an Ory session can be seen in the [Get session API](../../reference/api#tag/identity/operation/disableSession)
+response.
 
 | Property           | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 | :----------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `active`           | When set to `true`, the Ory Session is active and can be used to authenticate requests.                                                                                                                                                                                                                                                                                                                                                                                                                          |
 | `expires_at`       | Defines the time when the Session expires. This value depends on the session lifespan configuration.                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `authenticated_at` | Indicates the time of the most recent successful authentication. This value is updated when: <ul><li>The end-user authenticates with a second factor such as TOTP</li><li>The end-user refreshes their session using the [`/self-service/login/browser`](../../reference/api.mdx#operation/initializeSelfServiceLoginFlowForBrowsers) endpoint or [`/self-service/login/api`](../../reference/api.mdx#operation/initializeSelfServiceLoginFlowWithoutBrowser) endpoint and setting `refresh` to `true`</li></ul> |
 
-## Ory Session Cookie
+### Using Ory Session Cookie
 
 An Ory Session Cookie is issued when the user signs in through a web browser. To get the session payload, send a request to the
```

Comment:
Technically true, but we show a curl command below which is native using the cookie. I think maybe we should clarify or reword it to fit the narrative that cookies are meant for browser based flows (Server-side and single page).

This is also specific to which api is called e.g. `/login/browser` vs `/login/api`.

Reply:
The curl command example is only for developer simplicity to view on command line. I can add a note in about the appropriate usage